.github/workflows/exercises.yml

```diff
@@ -0,0 +1,42 @@
+name: Exercises
+on:
+  - push
+  - pull_request
+
+jobs:
+  job:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+    name: Rakudo Star on ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Rakudo Star with choco
+        if: runner.os == 'Windows'
+        run: >
+          choco install rakudostar;
+          echo "C:\rakudo\bin;C:\rakudo\share\perl6\site\bin"
+          | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      - name: Install Rakudo Star with brew
+        if: runner.os == 'macOS'
+        run: brew install rakudo-star
+
+      # https://github.com/Leont/getopt-long6/issues/10
+      - name: Install Getopt::Long:ver<0.1.7>
+        if: runner.os != 'Linux'
+        run: zef install https://github.com/Leont/getopt-long6.git@0.1.7
+
+      - name: Rakudo Star tests with prove6
+        if: runner.os != 'Linux'
+        run: prove6 --jobs 2 (Get-Item -Path exercises/*/.meta/solutions/)
+        shell: pwsh
```

Comment:
Interesting. The `pwsh` shell is also available on macOS? Nice.

Reply:
Bash and pwsh are the only shells supported on all three OS.